src/ControlzEx.Showcase/ControlzEx.Showcase.csproj

```diff
@@ -1,26 +1,38 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="MSBuild.Sdk.Extras">
+<Project Sdk="Microsoft.NET.Sdk.Wpf">
     <!-- Project properties -->
     <PropertyGroup>
+        <TargetFrameworks>net45;net462;netcoreapp3.0</TargetFrameworks>
```

Comment:
Why did you move this to the projects instead of relying on the value from Directory.Build.props?

Reply:
Good question: I noticed on my projects that I didn't get the build working, if the TargetFrameworks are specified in the Directory.Build.props. The same happened here, you get many weird errors!
I will actually address this to Microsoft in my next status email, maybe they can have a look it it.

P.S.
I now assume it has to do with the SDK "Microsoft.NET.Sdk.Wpf" vs "MSBuild.Sdk.Extras"...

Reply:
You might be right with your guess as it wasn't working in older versions of the Extras-SDK and was fixed some time ago. Maybe it's the same issue with the WPF-SDK.

Reply:
I think this is not something we can fix now, closing the conversation...